const tratarUncaughtException = function(){
    /*
    process.on('uncaughtException', function(error){
        console.log('ocorreu um erro.')
    })
    */
}

module.exports = {
    tratarUncaughtException
}